src/Gui/CommandLink.cpp

```diff
@@ -206,15 +206,19 @@ StdCmdLinkMake::StdCmdLinkMake()
 {
     sGroup        = "Link";
     sMenuText     = QT_TR_NOOP("Make link");
-    sToolTipText  = QT_TR_NOOP("Create a link to the selected object(s)");
+    sToolTipText  = QT_TR_NOOP("<p>An App::Link is a type of object that references or links"
```

Comment:
It's always cumbersome to put technical elements (here the HTML tags) into translatable text because as soon as a translator doesn't do it correctly the displayed string will be broken.

And I don't think we should mention App::Link in this text because it's a programming term and  a regular user shouldn't be bothered with such terms. Same with App::Part in the tooltip below.

Reply:
What is the proper way then? Without it the tooltip is going to be all in one line. Then you can put \n to break into lines, but it doesn't look as good and you have the same problem as with <p>.
I thought about using `sToolTipText  = "<p>" + QT_TR_NOOP("abc") + "</p>"` but this is not accepted.

Reply:
> I thought about using sToolTipText  = "<p>" + QT_TR_NOOP("abc") + "</p>" but this is not accepted.

This isn't possible. One way is using a static std::string and once constructed assign it to the sToolTipText
```
static std::string toolTip = ...
sToolTipText = toolTip.c_str();
```

Reply:
That still doesn't build. 'expression must have integral or unscoped enum type'
```
    static std::string toolTip = "<p>" + QT_TR_NOOP("A Link is a type of object that references or links"
        " to another object, in the same document, or in another document. Unlike Clones, Link"
        " references directly the original Shape, so it is more memory efficient which helps "
        "with the creation of complex assemblies from smaller subassemblies, and from multiple"
        " reusable components like screws, nuts, and similar fasteners.") + "</p>";
    sToolTipText  = toolTip.c_str();
```

Reply:
Ah of course, because all three parts are C strings and they cannot be concatenated. You have to write
`static std::string toolTip = std::string("<p>") + QT_TR_NOOP("A Link .... ") + "</p>";`

Reply:
Thanks, I moved the HTML out of the translation then.